documentation/components/Tabs.mdx

```diff
@@ -24,36 +26,67 @@ Tabs allow you to switch between different views of related information.
+
 <Playground>
-  <Component initialState={{}}>
-    {({ setState, state }) => (
-      <>
-      <Tabs name="defaultDiff" defaultOpen={3} items={[{header: 'Description', content: "Extra firm Seally snoring bed sets Americas top-rated brands bed sheets snooze medium latex encased coil bunk bed linens firm plush queen sleep expert dreams memory foam hypoallergenic free delivery 50% low price guarantee spine position pillowcases futon shut eye loft special financing brand ultra plush comfy save back pain sleep trial size zzzzzz king stomach innerspring Sterns & Foster adjustable bed Beautyrest box spring side bundle Purple malouf wrap dow"},{header: 'Specifications', content: 'Mattress ipsum dolor amet Comfy pillow protectors innerspring Serta twin queen hypoallergenic Seally king position malouf wrap mattress toppers crib snoring extra firm adjustable bed back free delivery futon 50% tulo mattress gel memory foam Sleepys medium shut eye california king headboards full hot bed frames cooling tulo mattress comfort pillow protectors comforters gel memory foam Serta twin hybrid Tempur-pedic Sleepys mattress toppers nap pillow top best prices crib customized shopping options soft neck snuggle'}, {header: 'About Beautyrest', content: 'Pillow top Sterns & Foster snoring pain Seally king gel memory foam bed sheets twin soft Purple pillowcases sleep trial save hot loft plush bundle latex brand low price guarantee box spring snooze sleep expert spine crib bed sets zzzzzz futon Sleepys comfort size comforters customized shopping options snuggle dreams bed frames Serta medium Beautyrest back encased coil pillow protectors tulo mattress down headboards hypoallergenic full position'}, {header: 'Delivery & Shipping', content: 'Tempur-pedic side neck ultra plush queen malouf wrap 50% california king best prices cooling linens stomach shut eye hybrid Americas top-rated brands firm memory foam free delivery adjustable bed bunk bed extra firm innerspring special financing mattress toppers comfy Serta encased coil gel memory foam medium customized shopping options comforters twin 50% zzzzzz position pain sleep expert linens stomach adjustable bed box spring Americas top-rated brands full Sterns & Foster king latex snooze side shut eye special financing'}, {header: 'Warranty', content: 'Mattress ipsum dolor amet Tempur-pedic Beautyrest brand snuggle crib pillow protectors snoring free delivery down sleep trial loft innerspring size bunk bed queen memory foam cooling dreams Purple neck Seally comfy california king tulo mattress back soft mattress toppers nap plush extra firm bed frames best prices comfort futon hypoallergenic firm hot bed sets malouf wrap hybrid pillowcases spine bed sheets save Sleepys pillow top headboards low price guarantee bundle ultra plush'} ]} />
-      </>
-    )}
-  </Component>
-</Playground>
\ No newline at end of file
+  {() => {
+    const items = [
+      {
+        header: 'Description',
```

Comment:
do you want to put tabData here too instead to condense?

Reply:
The reason I put it there was to show a working example in the website